### PR TITLE
E-1050 feat: add new fonts

### PR DESCRIPTION
**Pull request by ifiri**

### CODE REVIEW

**Comment by ifiri** [comment]
@dudiq ready for review

File: package.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
     "@endpass/class": "^0.20.1",
```
Comment (dudiq) [inline package.json:60]: can't see any declaration in `font-family` of new font

Comment (ifiri) [reply package.json:60]: here it's declared in `App.vue`

```
body,
input,
button,
select {
  font-family: 'Brutal Type', Helvetica, Arial, sans-serif;
}
```

all remains inherit font-family from `@endpass/ui`. When you will approve, you can check real font-family via devtools.

**Comment by ifiri** [comment]
@dudiq ready for approve

**Comment by kazurus** [comment]
@lamartire

**Comment by lamartire** [comment]
Don't works

**Comment by kazurus** [comment]
@ifiri

**Comment by ifiri** [comment]
@lamartire when and how? Worked for me

**Comment by ifiri** [comment]
@lamartire after investigations, found the problem. Now all should be ok. Also please run `yarn install` to be sure that you have latest `@endpass/ui` installed.

**Comment by kazurus** [comment]
@lamartire

**Pull request merged**